src/Col.js

```diff
@@ -5,22 +5,137 @@ import CustomPropTypes from './utils/CustomPropTypes';
 
 const Col = React.createClass({
   propTypes: {
+    /**
+     * The number of twelve available columns you wish to span
```

Comment:
The number is actually configurable, even though it's 12 by default. (See https://github.com/twbs/bootstrap/blob/master/less/variables.less#L324-L325) I would just say "The number of columns you wish to span for Extra small devices/Phones (<768px)"